lxfspacy.py

```diff
@@ -0,0 +1,37 @@
+import spacy
+
+language_dict = {
+  "pt": {
+    "speed": "pt_core_news_sm",
+    "accuracy": "pt_core_news_lg"
+  },
+  "en": {
+   "speed": "en_core_web_sm",
+   "accuracy": "en_core_web_trf"
+  },
+  "ru": {
+    "speed": "ru_core_news_sm",
+    "accuracy": "ru_core_news_lg"
+  }
+}
+ 
+def nlp(lang, opt):
+  return spacy.load(language_dict[lang][opt])
+
+#function getLexemes
+def getLexemes(txt, nlp):
+  renderedText = nlp(txt)
+  lexemeList = []
+  for token in renderedText:
+    if token.is_alpha:
+      lexemeList.append(token.lemma_)
+  return(lexemeList)
+
+def install_language_package(package_name):
```

Comment:
Let's adopt the following standard: functions are `camelCase` and variables are `snake_case`
```suggestion
def installLanguagePackage(package_name):
```